insight-hub/client.ts

```diff
@@ -370,9 +392,26 @@ export class InsightHubClient implements Client {
           if (!errorDetails) {
             throw new Error(`Error with ID ${args.errorId} not found in project ${project.id}.`);
           }
+          // Get the latest event for this error using the events endpoint with filters
+          let latestEvent = null;
+          try {
+            const eventsResponse = await this.errorsApi.listEventsOnProject(project.id, {
+              filters: {
+                "error": [{ type: "eq", value: args.errorId }],
+                ...args.filters
+              }
+            });
+            const events = eventsResponse.body || [];
+            latestEvent = events.length > 0 ? events[0] : null;
+          } catch (e) {
+            console.warn("Failed to fetch latest event:", e);
+            // Continue without latest event rather than failing the entire request
+          }
+
           const content = {
             error_details: errorDetails,
-            latest_event: (await this.errorsApi.viewLatestEventOnError(args.errorId)).body,
+            latest_event: latestEvent,
+            pivots: (await this.errorsApi.listErrorPivots(project.id, args.errorId, { ...(args.filters ? { filters: args.filters } : {}) })).body || [],
             url: await this.getErrorUrl(project, args.errorId),
```

Comment:
The URL won't be exactly correct if filters are applied – how much work would it be to plumb in the `toQueryString` function on the filters and use it here?

Reply:
Added the query string - I suppose it's just the filter it needs and not the other parameters like page size?